src/KnxDevice.h

```diff
@@ -272,7 +272,11 @@ inline void KnxDevice::SetDebugString(String *strPtr) {_debugStrPtr = strPtr;}
 #endif
 
 
-inline void KnxDevice::DebugInfo(const char comment[]) const
+inline void KnxDevice::DebugInfo(const char
+#if defined(KNXDEVICE_DEBUG_INFO)
+	       comment
+#endif
+	       []) const
```

Comment:
Again... why only add argument name when in debug mode? 

Looks like there is a hint that I do not know about?

pls. explain.

Reply:
see comment on src/KonnektingDevice.cpp below.
I agree it looks a bit ugly, but the variable is used only in debug mode. Compiling without debug will make the compiler warn about unused parameters.